src/test/scala/com/exasol/cloudetl/storage/StoragePropertiesTest.scala

```diff
@@ -50,6 +50,33 @@ class StoragePropertiesTest extends AnyFunSuite with BeforeAndAfterEach with Moc
     assert(BaseProperties(properties).getStoragePathScheme() === "s3a")
   }
 
+  test("getDeltaFormatLogStoreClassName returns storage class name for scheme") {
```

Comment:
```suggestion
  test("getDeltaFormatLogStoreClassName returns storage class name for schema") {
```

Reply:
It is path URI [scheme](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) and not database [schema](https://en.wikipedia.org/wiki/Database_schema).